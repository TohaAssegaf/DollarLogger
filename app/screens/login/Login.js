import styles from './styles'
import {
  HEADER_BACKGROUND_COLOR,
  HEADER_TEXT_COLOR,
} from '~/app/config/colors'
import React from 'react';
import { Text, View } from 'react-native'

export default class Login extends React.Component {
  static navigationOptions = ({ navigation}) => {
    return {
      title: 'Sign In',
      headerStyle: {
        backgroundColor: HEADER_BACKGROUND_COLOR,
      },
      headerTintColor: HEADER_TEXT_COLOR,
    }
  }
  render() {
    return (
      <View style={styles.screen}>
        <Text>Skeleton login screen</Text>
      </View>
    );
  }
}
