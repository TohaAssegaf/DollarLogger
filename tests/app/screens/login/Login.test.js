import Login from '~/app/screens/login/Login'
import React from 'react';
import renderer from 'react-test-renderer';
import configureMockStore from 'redux-mock-store'
import { Provider } from 'react-redux'
import thunk from 'redux-thunk'

const middlewares = [thunk]
const mockStore = configureMockStore(middlewares)

it('renders correctly', () => {
  const store = mockStore({})
  const rendered = renderer.create(<Provider store={store}><Login /></Provider>).toJSON();
  expect(rendered).toMatchSnapshot();
});
