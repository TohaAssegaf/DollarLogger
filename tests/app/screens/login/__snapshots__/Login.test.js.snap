// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "white",
      "flex": 1,
    }
  }
>
  <View>
    <RCTFBLoginButton
      onChange={[Function]}
      onLoginFinished={[Function]}
      style={
        Object {
          "height": 30,
          "width": 190,
        }
      }
    />
  </View>
</View>
`;
